README.md

```diff
@@ -1,89 +1,119 @@
 
 Get all of the files changed/modified in a pull request (`pull_request` or `pull_request_target`) or push's commits.
 You can choose to get all changed files, only added files, only modified files, only removed files, only renamed files, or all added and modified files.
 These outputs are available via the `steps` output context.
 The `steps` output context exposes the output names `all`, `added`, `modified`, `removed`, `renamed`, and `added_modified`.
 
-# Usage
+## Usage
 
 See [action.yml](action.yml)
 
 ```yaml
-- uses: Ana06/get-changed-files@v1.2
+- uses: Ana06/get-changed-files@v2.0.0
   with:
     # Format of the steps output context.
     # Can be 'space-delimited', 'csv', or 'json'.
     # Default: 'space-delimited'
     format: ''
     # Filter files using a regex
-    filter: '*'
+    glob-filter: '*'
 ```
 
-# Scenarios
+## Scenarios
 
-- [Get all changed files as space-delimited](#get-all-changed-files-as-space-delimited)
-- [Get all added and modified files as CSV](#get-all-added-and-modified-files-as-csv)
-- [Get all removed files as JSON](#get-all-removed-files-as-json)
+- [get-changed-files](#get-changed-files)
+  - [Get All Changed Files](#get-all-changed-files)
+  - [Usage](#usage)
+  - [Scenarios](#scenarios)
+    - [Get all changed files as space-delimited](#get-all-changed-files-as-space-delimited)
+    - [Get all changed *.yml files but exclude .github/*/*.yml files](#get-all-changed-yml-files-but-exclude-githubyml-files)
+    - [Get all added and modified files as CSV](#get-all-added-and-modified-files-as-csv)
+    - [Get all removed files as JSON](#get-all-removed-files-as-json)
+  - [Install, Build, Lint, Test, and Package](#install-build-lint-test-and-package)
+  - [License](#license)
 
-## Get all changed files as space-delimited
+### Get all changed files as space-delimited
 
 If there are any files with spaces in them, then this method won't work and the step will fail.
 Consider using one of the other formats if that's the case.
 
 ```yaml
 - id: files
-  uses: Ana06/get-changed-files@v1.2
+  uses: Ana06/get-changed-files@v2.0.0
   with:
-    filter: '*.php'
+    glob-filter: '*.php'
 - run: |
     for changed_file in ${{ steps.files.outputs.all }}; do
       echo "Do something with this ${changed_file}."
     done
 ```
 
-## Get all added and modified files as CSV
+### Get all changed *.yml files but exclude .github/*/*.yml files
+
+Be careful that the order of the glob has an importance.
+Therefore, including all YML files first and excluding the YML files of your `.github/*/` directories is the way to go to exclude them.
+If those two globs were inverted, you **would** include all the YML files, with the ones in your `.github/*/` directories.
+
+```yaml
+- uses: Ana06/get-changed-files@v2.0.0
+  with:
+    # Format of the steps output context.
+    # Can be 'space-delimited', 'csv', or 'json'.
+    # Default: 'space-delimited'
+    format: ''
+    # Filter files using a regex
+    glob-filter: |
+      *.yml
```

Comment:
@agraul not sure if you've seen this, but as GitHub actions core doesn't support a standard YAML as input, we have to declare the list a multi line string.

Reply:
Thanks for the ping, I missed that. It's a bit unfortunate, I didn't know that lists can't be passed, but as long as the structure is kept I think this is still nice!